srcpkgs/musescore/template

```diff
@@ -1,38 +1,33 @@
 # Template file for 'musescore'
-# TODO: Add MuseScore Soundfont as distfiles or separate package instead of
-# downloading in the build process.
-# https://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General/
 pkgname=musescore
-version=3.5.2
+version=3.6.2
 revision=1
 wrksrc="MuseScore-${version}"
 build_style=gnu-makefile
-make_build_args="PREFIX=/usr USE_SYSTEM_FREETYPE=ON MUSESCORE_BUILD_CONFIG=release"
+make_build_args="PREFIX=/usr USE_SYSTEM_FREETYPE=ON BUILD_WEBENGINE=OFF
+ DOWNLOAD_SOUNDFONT=OFF MUSESCORE_BUILD_CONFIG=release"
 make_install_args="-C build.release"
-hostmakedepends="cmake pkg-config"
+hostmakedepends="cmake pkg-config qt5-qmake qt5-host-tools qt5-tools"
 makedepends="jack-devel lame-devel libsndfile-devel portaudio-devel
  portmidi-devel pulseaudio-devel qt5-declarative-devel qt5-script-devel
  qt5-svg-devel qt5-tools-devel qt5-webchannel-devel
  qt5-location-devel qt5-xmlpatterns-devel qt5-plugin-mysql qt5-plugin-odbc
  qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-tds qt5-quickcontrols2-devel"
 depends="qt5-plugin-sqlite qt5-graphicaleffects qt5-quickcontrols2"
 short_desc="Create, play and print beautiful sheet music"
-maintainer="Brian Olsen <bnolsen@comcast.net>"
+maintainer="tibequadorian <tibequadorian@posteo.de>"
 license="GPL-2.0-or-later"
 homepage="https://musescore.org/"
-distfiles="https://github.com/musescore/MuseScore/archive/v${version}.tar.gz"
-checksum=75b8e783edca9d23718770b3f715d95b88f64545e0372f4ccba58ab073cc7068
-nocross="genManual: cannot execute binary file: Exec format error"
+_soundfont_url="https://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General"
+distfiles="https://github.com/musescore/MuseScore/archive/v${version}.tar.gz
+ ${_soundfont_url}/MuseScore_General.sf3
+ ${_soundfont_url}/MuseScore_General_License.md"
```

Comment:
Are there no versioned variants of this file? If upstream changes them, the template will just stop building :/

Reply:
I don't think so :/ These are the same that CMakeLists.txt uses to download ... at least they're checksummed now, remember https://github.com/void-linux/void-packages/pull/27451 ?

Reply:
so if upstream changes them, they would have to update too